Infrastructure/Models/Parcel.cs

```diff
@@ -7,11 +7,22 @@
 
 namespace Infrastructure.Models
 {
+    public enum Size
+    {
+        S, M, L, XL
+    }
     public class Parcel
     {
         public int Id { get; set; }
         [Key]
         public string? TrackingNumber { get; set; }
-        
+        public Size? Size { get; set; }
+
+        public int? ShipperID { get; set; }
+        public DateTime ShipmentDate { get; set; }
+        public Address? ShippingAddress { get; set; }
+
+        public DateTime DeliveryDate { get; set; }
+        public virtual Address? DeliveryAddress { get; set; }
```

Comment:
Why `virtual` used here? Because I don't see it use anywhere else where `Address` type is used.